api/dependencies.go

```diff
@@ -30,15 +30,15 @@ func InitDependencies(
 	conf Configuration,
 	dbPool *pgxpool.Pool,
 	signingKey *rsa.PrivateKey,
-	tokenVerifier tokenCookieVerifier,
+	optTokenVerifier ...tokenVerifier,
 ) dependencies {
 	database := postgres.New(dbPool)
 
-	if tokenVerifier == nil {
-		tokenVerifier = infra.InitializeFirebase(ctx)
+	if len(optTokenVerifier) == 0 {
+		optTokenVerifier = append(optTokenVerifier, infra.InitializeFirebase(ctx))
```

Comment:
I find this a litle bit too complicate for what it really does. IMO, either :
1. transform for an optionnal prop with default value (= no array)
```go
var tokenVerifier ...
if len(optTokenVerifier) >=  1 {
  tokenVerifier = optTokenVerifier[0
} else {
  tokenVerifier = infra.InitializeFirebase(ctx)
}
```
2. force the parameter to be defined, and use `infra.InitializeFirebase(ctx)` everywhere we use the "default" value (no more implicit default inside the implem)